test/deploy/kubernetes/provisioner-pods/deployment_control_node.yaml

```diff
@@ -33,37 +33,20 @@ spec:
   - name: cortx-control-node
     image: ghcr.io/seagate/cortx-all:2.0.0-latest-custom-ci
     imagePullPolicy: IfNotPresent
+    securityContext:
+      privileged: true
     env:
     - name: NODE_ID
       valueFrom:
         fieldRef:
           fieldPath: metadata.labels['id']
-    - name: CONFIG_INFO
-      value: "yaml:///etc/cortx/solution/config.yaml"
-    - name: CLUSTER_INFO
-      value: "yaml:///etc/cortx/solution/cluster.yaml"
-    - name: CORTX_CONFIG
-      value: "yaml:///etc/cortx/cluster.conf"
     args:
     - /bin/sh
     - -c
     - set -x;
-      export PATH=$PATH:/opt/seagate/cortx/provisioner/bin;
-      echo $NODE_ID > "/etc/machine-id";
-      cat /etc/machine-id;
-      cortx_setup config apply -f $CLUSTER_INFO -o $CORTX_CONFIG;
-      cortx_setup config apply -f $CONFIG_INFO -o $CORTX_CONFIG;
-      echo -e "#!/bin/bash\necho $*" > /usr/bin/motr_setup;
-      chmod +x /usr/bin/motr_setup;
-      cp -pf /usr/bin/motr_setup /usr/bin/s3_setup;
-      cp -pf /usr/bin/motr_setup /usr/bin/hare_setup;
-      cp -pf /usr/bin/motr_setup /usr/bin/csm_setup;
-      cp -pf /usr/bin/motr_setup /usr/bin/utils_setup;
-      cortx_setup cluster bootstrap -f  $CORTX_CONFIG;
-      ls /etc/cortx;
-      cat /etc/cortx/cluster.conf;
-      ls -l /etc/cortx/solution;
-      while true; do sleep 30; echo "Hello Kubernetes"; done;
+      chmod +x /opt/seagate/cortx/provisioner/bin/cortx_deploy;
```

Comment:
This should be controlled in the setup.py

Reply:
done